.github/pull_request_template.md

```diff
@@ -0,0 +1,75 @@
+<!--
+Hello, and thanks for contributing to imgix-core-js! 🎉🙌
+Please take a second to fill out PRs with the following template!
+-->
+
+## Description
+<!-- What is accomplished by this PR? If there is something potentially controversial in your PR, please take a moment to tell us about your choices. -->
+
+
+
+<!-- 
+Please use the checklist that is most closely related to your PR, and delete the other checklists. -->
+## Checklist: Fixing typos/Doc change
+
+- [ ] Each commit follows the conventional commit spec format: e.g. `chore(readme): fixed typo`. See the end of this file for more information.
+
+## Checklist: Bug Fix
+
+- [ ] Each commit follows the conventional commit spec format: e.g. `chore(readme): fixed typo`. See the end of this file for more information.
```

Comment:
nit: "conventional commit spec format" is a bit of a mouthful and "conventions" "spec," and "format" all mean roughly the same thing in this context. Really, we use this format because it's intended to be _"semantic."_

- "Each commit uses the **semantic format**, e.g. `chore(readme): fix typo`..."

Reply:
Just fyi, ["Conventional Commit"](https://www.conventionalcommits.org/en/v1.0.0/#summary) refers to the name of the spec -- but I see how that can be confusing. Maybe it would be better to include a link for users to learn more, if desired.

To your point, I think "[Conventional Commit](https://www.conventionalcommits.org/en/v1.0.0/#summary) format" would be a little more clear.

Reply:
Ah! TIL! Thanks for the link to "conventional commit" I was unaware of this (I assumed there was only semantic-release involvement here, but it's that + conventional commit, cool!)